src/main/scala/tscfg/generators/java/JavaGen.scala

```diff
@@ -167,17 +167,19 @@ class JavaGen(genOpts: GenOpts) extends Generator(genOpts) {
           }class $classNameAdjusted $extendsString{
              |  $classDeclMembersStr$classMemberGettersStr
              |  $membersStr
-             |  public $classNameAdjusted($ctorParams) {$superString
+             |  public $classNameAdjusted($ctorParams) {
+             |    $superString
```

Comment:
You have moved `$superString` to its own line, but this would generate an empty line in the output when there's no content captured there.  Although machine-generated, we have tried to generate code that is clean as much as practically possible.  Also, keep in mind that generated code is often also maintained under version control (I've seen this out there in the wild and in fact I do it myself), so an upgrade of the tscfg tool in client code would potentially generate unnecessary diffs just because of changes in spaces.  So, if you could review this kind of changes (and there might be other similar cases as with this `$superString`), that would be good. In general, please do no reformat any existing code that is unrelated with the PR/issue.

Reply:
But you know what? ...  actually, maybe the time is coming to start integrating a more intelligent, automatic, and consistent formatting of the output so one can just focus on the correct semantics of the output and let other tools to take care of the formatting.

So, I'd say, do not worry about these adjustments.  After the merge I'll look into integrating [scalafmt](https://scalameta.org/scalafmt/docs/installation.html#standalone-library) for Scala, and  probably [google-java-format](https://github.com/google/google-java-format#as-a-library) for Java.  Thanks.

Reply:
I just added all generated files to version control and pushed them into master.  Can you please merge that into your branch (you will need to first delete the ones you have on your side for the merge to proceed)?  Then, you can re-generate with your new code and use your favorite diff tool to see the differences.  To be clear, this is just to have an easier way to inspect changes in the generated code as the tool gets updated. Thanks,